Weather Alert/CoreData/City+CoreDataProperties.swift

```diff
@@ -16,5 +16,12 @@ extension City {
         return NSFetchRequest<City>(entityName: "City");
     }
 
+    @NSManaged public var deg: NSNumber?
+    @NSManaged public var name: String?
+    @NSManaged public var speed: NSNumber?
+    @NSManaged public var lon: NSNumber?
+    @NSManaged public var lat: NSNumber?
+    @NSManaged public var country: String?
+    @NSManaged public var id: NSNumber!
```

Comment:
@jackhoptz Just want to make sure you understand the usage of `!` for this `id` variable. As this comes from API - this means it's not guaranteed you'll get an `id`, which will cause the app to crash whenever `.id` is accessed throughout the app. I understand that by logic, there always must be an `id`, but I prefer better user experience without any crashes, even when the API is at fault. By using `?` in this case you'll force anyone that tries to access `id` to unwrap it correctly and present an error screen/pop-up if required.